### PR TITLE
added files

### PR DESCRIPTION
**Pull request by ElenaLadutska**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <header>Grocery List</header>
+  <div class="container">
+    <div class="typing-container">
+      <img src="./assets/imgonline-com-ua-Resize-4wYRb1IV1yU.png" alt="eraser" id="eraser">
+      <input type="text" id="text-input">
+    </div>
+
+    <div id="grocery-list"></div>
+  </div>
+</body>
```
Comment (kathrynowy) [inline index.html]: looks vague

File: index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const input = document.getElementById('text-input');
+let groceryList = document.getElementById('grocery-list');
```
Comment (kathrynowy) [inline index.js]: why let?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+input.addEventListener('keydown', event => {
+  if( event.code === 'NumpadEnter' || event.code === 'Enter') {
+    addGood();
+    input.value = '';
+  };
+});
+
+const addGood = () => {
+  let good = document.createElement('li');
```
Comment (kathrynowy) [inline index.js]: .

File: index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  good.className ='good-li';
+  good.textContent = input.value;
+  groceryList.append(good);
+
+  good.addEventListener('click', function() {
+    good.classList.toggle('through');
+    console.log(good.classList)
```
Comment (kathrynowy) [inline index.js]: ?

File: reset.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+blockquote:before, 
+blockquote:after,
+q:before, 
+q:after {
+	content: '';
+	content: none;
+}
+
+table {
+	border-collapse: collapse;
```
Comment (kathrynowy) [inline reset.css]: why 4 spaces here?

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ElenaLadutska) [reply index.html]: fixed

File: index.js (hunk 2 above)
Comment (ElenaLadutska) [reply index.js]: fixed. I forgot that we can add and remove children in const variable

File: index.js (hunk 4 above)
Comment (ElenaLadutska) [reply index.js]: removed. forgot to do it.

File: reset.css (hunk 5 above)
Comment (ElenaLadutska) [reply reset.css]: it looked like an arrow in vscode, I removed all of them and typed 2 spaces there

**Pull request merged**